src/Module/Contact/Profile.php

```diff
@@ -355,7 +355,7 @@ protected function content(array $request = []): string
 			'$archived'                  => ($contact['archive'] ? $this->t('Currently archived') : ''),
 			'$pending'                   => ($contact['pending'] ? $this->t('Awaiting connection acknowledge') : ''),
 			'$hidden'                    => ['hidden', $this->t('Hide this contact from others'), $localRelationship->hidden, $this->t('Replies/likes to your public posts <strong>may</strong> still be visible')],
-			'$notify'                    => ['notify', $this->t('Notification for new posts'), ($contact['notify_new_posts'] == 1), $this->t('Send a notification of every new post of this contact')],
+			'notify_new_posts'           => ['notify_new_posts', $this->t('Notification for new posts'), ($localRelationship->notifyNewPosts), $this->t('Send a notification of every new post of this contact')],
```

Comment:
Please respect the `$` notation for template variables, both work but every other key in the code is prefixed with a dollar sign.